lib/catalog/martyrology.ts

```diff
@@ -1372,6 +1440,19 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Florian',
       titles: [Title.Martyr],
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-floscel-martyr
+    // - https://nrm.m.wikipedia.org/wiki/Saint_Flosc%C3%A9%27
+    floscellus_of_normandy_martyr: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Floscellus',
+      titles: [Title.Martyr],
+      dateOfBirth: 280,
+      dateOfBirthIsApproximative: true,
+      dateOfDeath: 296,
```

Comment:
I could not find the death date in either of the three sources. I presume you have another source for it. :thinking:

Reply:
So I must have used the source from here, I think: https://nominis.cef.fr/contenus/saint/8225/Saint-Floscel.html

Reply:
Okay, then why not sure the full death date? :thinking:

```suggestion
      dateOfDeath: '296-09-17',
```